content/products.md

```diff
@@ -49,7 +49,7 @@ g3Entrance:
 
 <section class="g3-bg__mint">
   <div class="g3-outer-wrapper g3-flex-content g3-flex-content__reverse">
-    <div class="g3-col__65 g3-flex-content g3-mb-space__padding-lg-top">
+    <div class="g3-col__65 g3-flex-content g3-space__padding-md-top">
```

Comment:
Looks better with more padding on the top. I just noticed there's a white gap here in mobile version, and I think adding back `g3-mb-space__padding-lg-top` class will fix it...